LDAPAuthenticationProviderPlugin.cs

```diff
@@ -50,7 +50,7 @@ public async Task<ProviderAuthenticationResult> Authenticate(string username, st
                 if(ldapClient.Bound)
                 {
                     LdapSearchResults ldapUsers = ldapClient.Search(_config.LdapBaseDn,2,_config.LdapQuery,_attrs,false);
-                    if (ldapUsers == null || ldapUsers.Count == 0)
+                    if (ldapUsers == null)

```

Comment:
This is actually a bug (?) in how ldapClient.Search seems to work. It's returning before all the search results are returned from LDAP. Is there a way to do this synchronously, perhaps?

Reply:
Worked-round as per https://github.com/dsbenghe/Novell.Directory.Ldap.NETStandard/issues/4#issuecomment-277216636 in fa0ddcb.